recipes/blaze/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "blaze" %}
+{% set version = "0.11.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/blaze/blaze.git
+  gti_tag: {{ version }}
```

Comment:
I think `gti` is a typo, right? Can we use the tarball format instead? It saves us from cloning the repo.

Reply:
Ah, I didn't know that it worked for tags. Cool! I have updated the PR.